packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -1511,6 +1531,33 @@ class AccountInternal extends React.PureComponent {
     }
   };
 
+  onBatchDuplicate = async ids => {
+    this.setState({ workingHard: true });
+
+    let { data } = await runQuery(
+      q('transactions')
+        .filter({ id: { $oneof: ids } })
+        .select('*')
+        .options({ splits: 'grouped' })
+    );
+
+    let changes = {
+      added: data
+        .reduce((newTransactions, trans) => {
+          return newTransactions.concat(
+            realizeTempTransactions(ungroupTransaction(trans))
+          );
+        }, [])
+        .map(({ sort_order, ...trans }) => ({ ...trans }))
+    };
+
+    await send('transactions-batch-update', {
+      ...changes
+    });
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    await send('transactions-batch-update', changes);
```